src/main/java/core/basesyntax/utils/ParseDate.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.utils;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class ParseDate {
```

Comment:
```suggestion
public class DateParser/DateConverter {
```
для класів іменники вживаються зазвичай